README.md

```diff
@@ -7,7 +7,261 @@
 A Visual Studio Code extension for [ty](https://github.com/astral-sh/ty), an extremely fast
 Python type checker and language server, written in Rust.
 
-**This project is still in development and is not ready for production use.**
+> [!WARNING]
+>
+> This extension uses the preview version of ty and is not ready for production use.
+>
+> We're working hard to make ty stable and feature-complete, but until then, expect to encounter bugs,
+> missing features, and fatal errors.
+
+The extension ships with `ty==0.0.0a7`.
+
+## Features
+
+Currently, the extension supports the following features:
+
+- Showing diagnostics in the editor
+- Hover type information for symbols
+- Go to type definition
+- Inlay hints for assignment targets
+- [Completions](#completionsenable)
+
+> [!NOTE]
+>
+> If you want to test the language server features like completions, go to type definition, etc., it's
+> recommended to disable the language server from the [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) (if installed) by adding the
+> following [setting](https://code.visualstudio.com/docs/python/settings-reference#_intellisense-engine-settings) to your `settings.json`:
+>
+> ```json
+> {
+>   "python.languageServer": "None"
+> }
+> ```
+
```

Comment:
Another option is to introduce a `python.ty.disableLanguageService` option (similar to `python.pyrefly.disableLanguageService` to keep using ty exclusively for type checking and disable all language services like completions, goto definitions, etc.

Reply:
See: https://github.com/astral-sh/ty-vscode/issues/20